api/scripts/certification/next-gen/import-complementary-alone-feature-pilot-certification-centers-from-csv.js

```diff
@@ -87,7 +88,10 @@ async function main(filePath) {
 
     logger.info('Inserting pilot certification center ids in database... ');
     trx = await knex.transaction();
-    await trx('certification-center-features').del();
+    await trx('certification-center-features')
+      .del()
+      .leftJoin('features', 'features.id', 'certification-center-features.featureId')
+      .where('features.key', CERTIFICATION_FEATURES.CAN_REGISTER_FOR_A_COMPLEMENTARY_CERTIFICATION_ALONE.key);
```

Comment:
Neatpick : Je mettrais le del en dernier dans l'ordre de lecture (je sélectionne et j'applique l'action à la sélection)

```suggestion
    await trx('certification-center-features')
      .leftJoin('features', 'features.id', 'certification-center-features.featureId')
      .where('features.key', CERTIFICATION_FEATURES.CAN_REGISTER_FOR_A_COMPLEMENTARY_CERTIFICATION_ALONE.key)
      .del();
```

Reply:
Done